bin/node/src/commands/bundled.rs

```diff
@@ -131,11 +138,13 @@ impl BundledCommand {
         }
     }
 
+    #[allow(clippy::too_many_arguments)]
     async fn start(
         rpc_url: Url,
         data_directory: PathBuf,
         batch_prover_url: Option<Url>,
         block_prover_url: Option<Url>,
+        ntx_proving_service_address: Option<Url>,
```

Comment:
nit: I think we can keep this named `tx_prover_url` (or at least change `_address` to `_url` for consistency).

Also, not for this PR, but it may make sense to group prover configs into a dedicated struct - e.g.,:
```rust
pub struct ProverConfig {
    tx_prover_url: Option<Url>,
    batch_prover_url: Option<Url>,
    block_prover_url: Option<Url>,
}
```